scripts/build-package.ts

```diff
@@ -46,12 +46,22 @@ async function buildEsm({
 }) {
 	const res = await build({
 		entryPoints: sourceFiles,
-		outdir: path.join(sourcePackageDir, 'dist'),
-		bundle: false,
+		outdir: path.join(sourcePackageDir, 'dist/esm'),
+		bundle: true,
```

Comment:
would rather not have our output bundled, to make for easier debugging in situ.

Reply:
That was my first thought, too, but that proposed workaround (https://github.com/evanw/esbuild/issues/622#issuecomment-769462611) for the file extension doesn't work without it, but then I checked the `dist` folder after building and I saw that each file was still there and not bundled into a single one.
<img width="335" alt="image" src="https://user-images.githubusercontent.com/60748911/225931975-7e5617b4-27b4-4c10-9ca0-31180242923d.png">

So hopefully that's sufficient?

Reply:
ah i see, it marks the files as external to prevent bundling. it's fine then 👍🏼